app-config.yaml

```
@@ -109,6 +109,7 @@ catalog:
 
 gitops:
   baseUrl: ${WEAVE_GITOPS_URL}
+  #  readOnly: false # Set to true to disable sync/suspend/resume of resources
```

```suggestion
  # readOnly: true # Set to true to disable sync/suspend/resume of resources
```

Just to uncomment and go